tests/integration/kubernetes/k8s-policy-pvc.bats

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2024 Microsoft.
```

Comment:
I vaguely remember that reviewers directed me to change Intel -> Microsoft for a similar change. So, I suggest changing this line too.

Reply:
Changed to Intel. The other policy tests still say Microsoft, though.